client-src/elements/chromedash-admin-blink-page.ts

```diff
@@ -62,29 +69,21 @@ export class ChromedashAdminBlinkPage extends LitElement {
       `,
     ];
   }
-  static get properties() {
-    return {
-      loading: {type: Boolean},
-      user: {type: Object},
-      _editMode: {type: Boolean},
-    };
-  }
-
-  /** @type {import('chromestatus-openapi').DefaultApiInterface} */
-  _client;
 
-  /** @type {Array<import('chromestatus-openapi').OwnersAndSubscribersOfComponent>} */
+  @property({type: Boolean})
+  loading = true;
+  @property({type: Object})
+  user = null;
+  @property({type: Boolean})
+  _editMode = false;
+  @property({
+    type: Array<OwnersAndSubscribersOfComponent>,
+  })
   components;
-
-  /** @type {Map<int, import('chromestatus-openapi').ComponentsUser}> */
+  @property({type: Map<number, ComponentsUser>})
   usersMap;
```

Comment:
Similar to `components`, but without the `|undefined`
```suggestion
  @state()
  usersMap!: Map<number, ComponentsUser>;
```